src/main/java/core/basesyntax/strategyimpl/BalanceHandler.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.strategyimpl;
+
+import core.basesyntax.storage.Storage;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceHandler implements OperationHandler {
+    @Override
+    public void calculateValue(String fruit, int quantity) {
+        if (Storage.fruits.containsKey(fruit)) {
+            Storage.fruits.put(fruit, Storage.fruits.get(fruit) + quantity);
+            return;
```

Comment:
IMHO
Maybe I dont get the Idea of the Balance strategy but I thought that this is starting quantity - so it should just add quantity without checks, 
but if I wrong than you should change return - let it return new Hashmap from storage